src/components/engagement/events/engagement-created.event.ts

```diff
@@ -0,0 +1,11 @@
+import { ISession } from '../../../common';
+import { CreateCeremony } from '../../ceremony/dto';
+import { LanguageEngagement } from '../dto';
+
+export class EngagementCreatedEvent {
+  constructor(
+    readonly languageEngagement: LanguageEngagement,
+    readonly input: CreateCeremony,
```

Comment:
An engagement being created has nothing to do with a ceremony as far as the engagement module is concerned. That's the whole point of this task.

Reply:
Removed Ceremony.